gobrew.go

```diff
@@ -154,14 +159,15 @@ func (gb *GoBrew) Interactive(ask bool) {
 	}
 
 	if modVersion != NoneVersion && currentMajorVersion != modVersion {
-		color.Warnf("GO Installed Version (%s) and go.mod Version (%s) are different.\n", currentMajorVersion, modVersion)
-		c := true
-		if ask {
-			c = askForConfirmation("Do you want to use GO version same as go.mod version (" + modVersion + "@latest)?")
-		}
-		if c {
-			gb.Use(modVersion + "@latest")
-		}
+		color.Warnf("⚠️  GO Installed Version (%s) and go.mod Version (%s) are different.\n", currentMajorVersion, modVersion)
+		fmt.Println("   Please consider updating your go.mod file")
+		// c := true
```

Comment:
Why are we leaving this comment if the code will not be used?

Reply:
Fixed